composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle" : "~2.3",
-        "fsi/admin-bundle" : "1.1.x-dev"
+        "symfony/framework-bundle" : "^2.3",
+        "fsi/admin-bundle" : "~1.1@dev"
```

Comment:
when we tag `1.1.0` then:
- we justify bc breaks we've made so far
- we should change dependencies in other bundles to `~1.1` and then change it again to `~2.0` after tagging `2.0.0` unless we won't tag `2.0.0` at all